job/server/src/main/java/alluxio/worker/job/task/TaskExecutorManager.java

```diff
@@ -71,20 +68,13 @@ public TaskExecutorManager(int taskExecutorPoolSize) {
    * @param taskId the task id
    * @param result the task execution result
    */
-  public synchronized void notifyTaskCompletion(long jobId, int taskId, Object result) {
+  public synchronized void notifyTaskCompletion(long jobId, int taskId, Serializable result) {
```

Comment:
Do any callers need to be updated?

Reply:
Nope. It was already passing in a Serializable